starboard/shared/starboard/player/filter/testing/test_util.h

```diff
@@ -39,9 +39,11 @@ namespace testing {
 
 typedef std::tuple<const char*, SbPlayerOutputMode> VideoTestParam;
 
-enum HeaacOption {
+enum DmpFileOption {
   kIncludeHeaac,
   kExcludeHeaac,
+  kIncludeHeaacExcludeSintel,
```

Comment:
Having "Sintel" in the enum name is a little too specific, if we remove the Sintel files in the name we'd have to change it. Or if we decide to add stereo Sintel files, we'd also have to change it.

I'm still in favor of adding a new field in `GetSupportedAudioTestFiles()` to indicate if passthrough files should be used.